cloud-formation/membership-app.cf.json

```diff
@@ -108,9 +108,9 @@
 
                         "adduser --system --home /membership --disabled-password membership",
 
-                        {"Fn::Join":["", ["aws --region ", { "Ref": "AWS::Region" }, " s3 cp s3://membership-dist/", { "Ref" : "Stack" }, "/", { "Ref" : "Stage" }, "/frontend/app.zip /tmp"]]},
+                        {"Fn::Join":["", ["aws --region ", { "Ref": "AWS::Region" }, " s3 cp s3://membership-dist/", { "Ref" : "Stack" }, "/", { "Ref" : "Stage" }, "/frontend/frontend-1.0-SNAPSHOT.tgz/tmp"]]},
 
-                        "unzip -d /membership /tmp/app.zip",
+                        "unzip -d /membership /tmp/frontend-1.0-SNAPSHOT.tgz",
```

Comment:
Good catch! I still don't know how to make it generate a zip file rather than a tgz (so I still need a name change), but I should have remembered that I also need a change of decompression command.